test/doctest/theme-custom.html

```diff
@@ -133,39 +165,37 @@ <h2>Actually, they are…​</h2>
     parallaxBackgroundVertical: null,
     // The display mode that will be used to show slides
     display: 'block',
-
+  
```

Comment:
can we remove the trailing spaces?

Reply:
Sure, `rake doctest:generate` creates them and I was too lazy to remove them. If `doctest` doesn't mind (and I my guess is it doesn't) then I'll remove them.

Reply:
Fixed in e5b8b64